tests/acceptance/features/apiGraph/deleteGroup.feature

```diff
@@ -0,0 +1,68 @@
+@api @skipOnOcV10
+Feature: delete groups
+  As an admin
+  I want to be able to delete groups
+  So that I can remove unnecessary groups
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And the administrator has given "Alice" the role "Admin" using the settings api
+
+
+  Scenario Outline: admin user deletes a group
+    Given group "<group_id>" has been created
+    When user "Alice" deletes group "<group_id>" using the Graph API
+    And the HTTP status code should be "204"
+    And group "<group_id>" should not exist
+    Examples:
+      | group_id            | comment                               |
```

Comment:
is this comment for `group-id` name?

Reply:
I think it's only used to give more info to the group name being created